orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Execution.java

```diff
@@ -168,7 +165,7 @@ public void setKeepWaitingPipelines(boolean keepWaitingPipelines) {
 
   private final List<Stage> stages = new ArrayList<>();
 
-  @JsonManagedReference
+  @JsonIgnoreProperties(value = "execution")
```

Comment:
Can you drop a comment here that this field is a `JsonBackReference`? Does this ignore `stage.execution`? Maybe clarify that in a comment as well.

Reply:
Done.